paper-slider.html

```diff
@@ -168,15 +165,10 @@
       }
 
       #sliderKnob {
-        @apply(--layout-center-justified);
-        @apply(--layout-center);
-        @apply(--layout-horizontal);
-
         position: absolute;
         left: 0;
-        top: 0;
-        width: 32px;
-        height: 32px;
```

Comment:
For a11y purposes, a ripple shows up if the slider received keyboard focus. (e.g. after pressing tab). That ripple takes the full size of the parent container which in this case is `#sliderKnob`.  If the width and/or height is not set in the parent, the ripple won't show up anymore or it will take the wrong size.